axelor-account/src/main/resources/domains/Invoice.xml

```diff
@@ -155,6 +155,12 @@
 	    public static final int OPERATION_SUB_TYPE_DEFAULT = 1;
 	    public static final int OPERATION_SUB_TYPE_ADVANCE = 2;
 	    public static final int OPERATION_SUB_TYPE_BALANCE = 3;
+	    
+	    // REPORTS INVOICE TYPE
+		public static final int INVOICE_TYPE_PROFORMA = 1;
+		public static final int INVOICE_TYPE_ORIGINAL_INVOICE = 2;
```

Comment:
Change 'INVOICE_TYPE_' to 'REPORT_TYPE_'. Change method parameters and variable names according to it.